docs/sources/trackers/google-tag-manager/snowplow-template/plugins/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Plugins"
+sidebar_position: 200
+---
+
+Plugins provide an easy way to distribute custom behavior for the tracker.
```

Comment:
I didn't understand initially that these are the existing JS tracker plugins, add an extra sentence or two of intro please

Reply:
They _sort of_ are, but many of the existing plugins don't work out the box with GTM due to the way GTM sandboxes the JS (e.g. attempting to select a DOM element will just return nothing). I've clarified a bit, can add a bit more if it's still confusing.